src/Compilers/Core/Portable/SourceGeneration/GeneratorDriverOptions.cs

```diff
@@ -40,7 +48,7 @@ public GeneratorDriverOptions(IncrementalGeneratorOutputKind disabledOutputs, bo
         /// <param name="trackIncrementalGeneratorSteps"></param>
         /// <param name="baseDirectory">Absolute path to the base directory used for file paths of generated files.</param>
         /// <exception cref="ArgumentException"><paramref name="baseDirectory"/> is not an absolute path.</exception>
-        public GeneratorDriverOptions(IncrementalGeneratorOutputKind disabledOutputs = IncrementalGeneratorOutputKind.None, bool trackIncrementalGeneratorSteps = false, string? baseDirectory = null)
+        public GeneratorDriverOptions(IncrementalGeneratorOutputKind disabledOutputs = IncrementalGeneratorOutputKind.None, bool trackIncrementalGeneratorSteps = false, string? baseDirectory = null, string? projectName = null)
```

Comment:
@333fred Do we need to go thru an API review meeting for this, or is it ok to just do it via mail?

Reply:
Think an email is fine.